pkg/catalogv2/system/system.go

```diff
@@ -317,7 +342,7 @@ func (m *Manager) waitPodDone(op *catalog.Operation) error {
 		return nil
 	}
 
-	sec := int64(60)
+	sec := int64(300)
```

Comment:
This change to 5 minutes seems like a long. If someone installs 50+ charts and they all wait this long on the operation, there may be blocking issues due to limited workers. Is this needed to resolve the issue with pods being spammed?

Reply:
That is just the timeout for the watcher, it won't necessarily wait that long. The reason behind that change is that the operation pod itself has a timeout of 5 minutes, so if the watcher ends before that, the operation may end successfully but the watcher would have returned an error and the chart wouldn't have been added to the desired charts map.

There is a deeper problem there in that the waitPodDone function is blocking the goroutine the manager struct is on. I created an issue for that so we can take a look at it later

Reply:
I don't think its advisable to extend the duration of blocking until that is fixed.

Reply:
I agree with that. What do you think @rohitsakala?

Reply:
Sure @diogoasouza